core/common.ts

```diff
@@ -86,38 +91,58 @@ export function setMainWorkspace(workspace: Workspace) {
 }
 
 /**
- * Currently selected copyable object.
+ * Returns the current selection.
  */
-let selected: ISelectable | null = null;
+export function getSelected(): ISelectable | null {
+  const focused = getFocusManager().getFocusedNode();
+  if (focused && isSelectable(focused)) return focused;
+  return null;
+}
 
 /**
- * Returns the currently selected copyable object.
+ * @internal
  */
```

Comment:
Add a comment: returns the selected block, or the first non-shadow parent if the selected block was a shadow.

Reply:
Thanks for the reminder--I definitely will add a comment. Just added the '@internal' to make it clear for draft review.

Reply:
This has been addressed--marking as resolved to keep open only those threads that may need additional discussion.